.test-infra/jenkins/job_PerformanceTests_FileBasedIO_IT.groovy

```diff
@@ -129,16 +129,15 @@ private void create_filebasedio_performance_test_job(testConfiguration) {
         }
 
         def argMap = [
-                benchmarks               : 'beam_integration_benchmark',
-                beam_it_timeout          : '1200',
-                beam_it_profile          : 'io-it',
-                beam_prebuilt            : 'false',
-                beam_sdk                 : 'java',
-                beam_it_module           : 'sdks/java/io/file-based-io-tests',
-                beam_it_class            : testConfiguration.itClass,
-                beam_it_options          : common_job_properties.joinPipelineOptions(pipelineOptions),
-                beam_extra_mvn_properties: '["filesystem=gcs"]',
-                bigquery_table           : testConfiguration.bqTable,
+                benchmarks           : 'beam_integration_benchmark',
```

Comment:
Minor question, is this reformatting automatic? I kind of dislike having everything aligned to colons but it seems this is common in groovy.

Reply:
It is automatic.